scripts/articles.js

```diff
@@ -18,51 +17,52 @@ Article.prototype.postAge = function(date) {
   var m2 = parseInt(date.slice(5,7));
   var y2 = parseInt(date.slice(0,4));
   return Math.round(Math.abs((new Date(y2,m2,d2).getTime() - new Date(y1,m1,d1).getTime())/(24*60*60*1000)));
-}
+};
 // Article method to display a blog post to the DOM
 Article.prototype.toHTML = function () {
-  // console.log(this);
+  console.log('-> Article.toHTML');
   var age = this.postAge(this.publishedOn);
   var html = this.handlebarTest(this);
-  // console.log(html);
   $('#app').append(html);
   return html;
-}
+};
 
 Article.prototype.insertRecord = function(callback) {
+  console.log('-> Article.insertRecord');
   // insert article record into database
-  webDB.insertRecord(this);
-  // webDB.execute(
-  //   // TODO: Add SQL here...
-  //   {
-  //     'sql':'INSERT INTO articles (author, title, authorUrl, publishedOn, category, markdown)'}]
-  //
-  //    ,
-  //   callback
-  // );
+  // webDB.insertRecord(this);
+  webDB.execute(
+    [
+      {
+        'sql': 'INSERT INTO articles (title, author, authorUrl, category, publishedOn, markdown) VALUES (?, ?, ?, ?, ?, ?);',
+        'data': [this.title, this.author, this.authorUrl, this.category, this.publishedOn, this.markdown]
+      }
+    ],
+    callback
+  );
 };
 Article.prototype.updateRecord = function(callback) {
+  console.log('-> Article.updateRecord');
   //update article record in databse
   webDB.execute(
-    "UPDATE articles SET title='" + this.title + "', author='" + this.author + "', authorUrl='" + this.authorUrl + "', category='" + this.category + "', publishedOn='" + this.publishedOn + "', markdown='" + this.markdown + "' WHERE id='" + this.id + "';"
+    'UPDATE articles SET title="' + this.title + '", author="' + this.author + '", authorUrl="' + this.authorUrl + '", category="' + this.category + '", publishedOn="' + this.publishedOn + '", markdown="' + this.markdown + '" WHERE id="' + this.id + '";'
```

Comment:
When possible, use the (?) placeholder technique used in the insertRecord function instead of concatenating a SQL statement. It's much safer. Remember, the article properties originally come from user input. We don't want to enable a SQL injection attack.